src/js/main.js

```diff
@@ -124,17 +124,17 @@ class Site {
       const day = $('#birthday-day').val();
       const year = $('#birthday-year').val();
       if ( month == null || month == "" || day == null || day == "" || year == null || year == "") {
-        alert('Please fill out all fields.');
+        $('.error-rules').replaceWith("<p>Please fill out all fields</p>");
```

Comment:
call the element `#age-form-response` and reference it that way. `replaceWith` is a dangerous function because you're adding and removing elements from the DOM. You don't need to replace the entire element, just it's contents. I sent you references for `html()` and `text()` on Slack.